docs/index.html

```diff
@@ -48,7 +48,7 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Asks for help when they need it.</li>
         <li>Supports code in production, even if they did not write it.</li>
         <li>Can feel like an impostor at times.</li>
-        <li>Makes sure discussions are always productive and everyone gets to have their say before the team makes a decision. Only disregards opinions by providing arguments.</li>
+        <li>Prefers when discussions are productive and everyone gets to have their say before the team makes a decision. Only disregards opinions by providing arguments.</li>
```

Comment:
+1 to the intent. 

> Only disregards opinions by providing arguments.

While we're here, maybe remove this strict assertion as well?